apis/nucleus/src/components/Cell.jsx

```diff
@@ -297,6 +297,19 @@ const Cell = forwardRef(
       },
     });
 
+    let bgColor = 'transparent';
+
+    if (
+      layout &&
+      layout.components &&
+      layout.components.find((comp) => comp.key === 'general') &&
+      layout.components.find((comp) => comp.key === 'general').bgColor &&
+      layout.components.find((comp) => comp.key === 'general').bgColor.color &&
+      layout.components.find((comp) => comp.key === 'general').bgColor.color.color
```

Comment:
simplify this:
const component = layout?.components?.find((comp) => comp.key === 'general');
if(component?.bgColor?.color) {
...
}
I think you can use the ? notation here, but if not you can at least stop it from running .find() 4 times!

Reply:
The thing is that for some reason, lint is not allowing to commit with the ? notation, it's throwing the following error.
 
<img width="521" alt="Screenshot 2022-09-06 at 3 12 05 PM" src="https://user-images.githubusercontent.com/101629580/188602824-4a55e56e-238d-4337-8465-ce015e119c44.png">

I had done the same thing in a single line earlier, but it didn't allow me to commit

Reply:
ok, but you can atlest simplify it to:
const component = layout &&  layout.components ? layout.components.find((comp) => comp.key === 'general') : null;

Reply:
ok will do that